### ISSUE TITLE
bad use of TCHAR_TO_UTF8 macros

### ISSUE THREAD

**Issue by joeGraf** [gaslightgames/UE4_EOS_Plugin#36]
Credentials.Id = TCHAR_TO_UTF8( *AccountCredentials.Id );
out of scope here some memory is freed
				Credentials.Token = TCHAR_TO_UTF8( *AccountCredentials.Token );
out of scope here some memory is freed

the buffers from these macros are only valid for their scope, which terminates at the ;

their use as parameters to functions is fine because the scope ends upon return from the call. 

strcpy(DestUTF8, TCHAR_TO_UTF8( *AccountCredentials.Token ));  <-- VALID

Credentials.Token = TCHAR_TO_UTF8( *AccountCredentials.Token );  <-- CRASH

**Comment by fastlorisstudios** [comment]
Awesome, thank you!

Quick search of the Engine source, to find an historic way of allocating the correct amount of space for "DestUTF8" and I have this:
```
std::string IdUTF8( TCHAR_TO_UTF8( *AccountCredentials.Id ) );
std::string TokenUTF8( TCHAR_TO_UTF8( *AccountCredentials.Token ) );
Credentials.Id = IdUTF8.c_str();
Credentials.Token = TokenUTF8.c_str();
```

**Comment by fastlorisstudios** [comment]
Fixed in a493cb3
